MicrosoftStoreServicesSample/Controllers/PurchaseController.cs

```diff
@@ -450,7 +513,6 @@ private static string FormatResponseFromClawbackActionList(List<ClawbackActionIt
         {
             var response = new StringBuilder("");
             response.Append(
-                    "|--------------------------------------------------------------------------------------------------------------------------------------------------------------------|\n" +
                     "| Action State | ProductId    | Quantity | LineItemState | Purchased Date         | Refunded Date          | Candidate Id     | Candidate Consume Date | LineItemId                           |\n" +

```

Comment:
Not part of this change, but you may want to consider switching to AppendFormat using width operators `{0,20}` and such...

https://docs.microsoft.com/en-us/dotnet/standard/base-types/composite-formatting#alignment-component